### PR TITLE
Detect furry scum

### PR DESCRIPTION
**Pull request by T-117**
Detects incoming furry scum and tells them where they can do with themselves.

### CODE REVIEW

File: src/services/SubmissionService.ts [hunk 1]
```diff
@@ -250,8 +250,15 @@ export class SubmissionService implements OnInit {
         if (!customWad && !entry.wadURL) {
             throw new Error("Either WAD URL or a file must be uploaded.");
         }
+
+        const furryDetectors = ["furry", "yiff", "yarf", "yerf", "yowl", "uwu"];
         const submitterName = entry.submitterName;
         const email = entry.submitterEmail;
+        for (const detector of furryDetectors) {
+            if (submitterName.toLowerCase().includes(dector) || email.toLowerCase().includes(detector) || entry.wadName.toLowerCase().includes(detector) {
```
Comment (VictoriqueMoe) [inline src/services/SubmissionService.ts:258]: submitterName can be null. 

use `(submitterName?.toLowerCase().includes(dector)`

Comment (T-117) [reply src/services/SubmissionService.ts:258]: good catch

File: src/services/SubmissionService.ts [hunk 2]
```diff
@@ -250,8 +250,15 @@ export class SubmissionService implements OnInit {
         if (!customWad && !entry.wadURL) {
             throw new Error("Either WAD URL or a file must be uploaded.");
         }
+
+        const furryDetectors = ["furry", "yiff", "yarf", "yerf", "yowl", "uwu"];
         const submitterName = entry.submitterName;
         const email = entry.submitterEmail;
+        for (const detector of furryDetectors) {
+            if (submitterName?.toLowerCase().includes(detector) || email.toLowerCase().includes(detector) || entry.wadName.toLowerCase().includes(detector) {
```
Comment (VictoriqueMoe) [inline src/services/SubmissionService.ts:258]: you are missing an end bracket:

**Review by VictoriqueMoe — CHANGES_REQUESTED** [review]

**Review by VictoriqueMoe — APPROVED** [review]

**Comment by VictoriqueMoe** [comment]
not fallproof enough. need to detect a furry from smell instead

**Comment by T-117** [comment]
sad.

**Pull request closed without merging**